packages/mobile/src/assets/images/iconWand.svg

```diff
@@ -1,6 +1,6 @@
+        <path d="M42.0732944,4.44132549 L43.5922028,5.95945415 C44.0634752,6.42851435 44.0840533,7.18475778 43.6389864,7.6787524 L36.4749733,15.6197419 L36.4756336,17.8011696 C36.4756336,18.4471167 35.9847177,18.9784042 35.3556263,19.0422919 L35.2280702,19.0487329 L33.3840156,19.0487329 L35.1339733,17.1057419 L11.6148148,43.1766082 C10.6907926,44.196726 9.11522725,44.2759229 8.09356721,43.3536063 C8.06237813,43.3255361 8.03196877,43.2966862 8.00233914,43.2670566 L4.7695906,40.0366472 C3.79543426,39.0622828 3.79543426,37.4827466 4.7695906,36.5083821 C4.79922022,36.4787525 4.82962958,36.4499026 4.86003894,36.4218324 L14.0189733,28.1587419 L14.0194932,25.2865497 C14.0194932,24.6406026 14.510409,24.1093151 15.1395004,24.0454274 L15.2670565,24.0389864 L18.5859733,24.0387419 L40.3578948,4.3984405 C40.8503066,3.95480021 41.6036659,3.97363419 42.0732944,4.44132549 Z M37.7231969,26.5341131 L39.8019494,30.6923977 L43.9610137,32.7719298 L39.8019494,34.851462 L37.7231969,39.0097466 L35.6444445,34.851462 L31.4853801,32.7719298 L35.6444445,30.6923977 L37.7231969,26.5341131 Z M10.2768031,4.07797266 L12.3555555,8.23625727 L16.5146199,10.3157894 L12.3555555,12.3953216 L10.2768031,16.5536062 L8.19805064,12.3953216 L4.0389863,10.3157894 L8.19805064,8.237037 L10.2768031,4.07797266 Z M21.5048733,4.07797266 L22.7524366,6.57309937 L25.2475634,7.82066273 L22.7524366,9.06822608 L21.5048733,11.5633528 L20.2573099,9.06822608 L17.7621832,7.82066273 L20.2573099,6.57309937 L21.5048733,4.07797266 Z" id="Combined-Shape" fill="#000"></path>
```

Comment:
what changed here?

Reply:
fill=#000 wasn't specified so it was not actually using style-system fill